src/helpers/generateFieldType.ts

```diff
@@ -52,6 +52,19 @@ export const postgresqlTypeMap: TypeMap = {
   Unsupported: "unknown",
 };
 
+export const sqlServerTypeMap: TypeMap = {
+  BigInt: "number",
+  Boolean: "boolean",
+  Bytes: "Buffer",
+  DateTime: "Timestamp",
+  Decimal: "string",
```

Comment:
The same as in #80 

```suggestion
  Decimal: "number",
```

Reply:
@fehnomenal I've closed #80 due to default driver behavior. I'm going to say that this change is not needed for now but I'll add a dialect test later to be sure.

Reply:
Well, alright I guess you're right. In the meantime I noticed that I can override the generated types via the config.